presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -151,6 +151,11 @@ public static QueryRunner createNativeQueryRunner(
             boolean addStorageFormatToPath)
             throws Exception
     {
+        ImmutableMap.Builder<String, String> hivePropertiesBuilder = new ImmutableMap.Builder<>();
```

Comment:
nit: maybe

```
ImmutableMap.<String, String>builder()
   .putAll(getNativeWorkerHiveProperties(storageFormat))
   .put("hive.allow-drop-table", "true")
   .build();
```

and let's add comment to explain that we need hive.allow-drop-table=true because we use "legacy" security.

Reply:
@mbasmanova Thanks for the review, added the comment .